spec/controllers/api/v3/dashboards/chart_data_controller_spec.rb

```diff
@@ -38,5 +38,30 @@
       data = json['data']
       expect(data.map { |e| e['x'] }).to eq([2015])
     end
+
+    it 'returns values for multiple matched companies' do
```

Comment:
This test is fine, but it tests what used to work previously, which is filtering by importer and exporter node; what we should add is another test which tests filtering by 2 exporters or 2 importers, as this is the functionality that's new.